release-notes.md

```diff
@@ -1,11 +1,19 @@
 ## Release Notes
 
+### SimpleBatchUpload 2.0.2
+
+Released on June 19, 2025.
+
+* Improved support for MediaWiki 1.41 and above
```

Comment:
We didn't do any improvements for that. The other fixes for CI and minimal code cleanup probably isn't worth mentioning on release notes.